pkg/server/endpoints/authorized_entryfetcher_registration_entries.go

```diff
@@ -156,6 +158,10 @@ func (a *registrationEntries) updateCache(ctx context.Context) error {
 	return nil
 }
 
+// missedStartupEvents will check for any events come in with an ID less than the first event ID we receive.
+// For example if the first event ID we receive is 3, this function will check for any IDs less than that.
+// If event ID 2 comes in later on, due to a long running transaction, this function will update the cache
+// with the information from this event. This function will run the first sqlTransactionTimeout hours after startup.
```

Comment:
nit: sqlTransactionTimeout is not a unit of hours
```suggestion
// with the information from this event. This function will run until time equal to sqlTransactionTimeout has elapsed after startup.
```